protostar/composition_root.py

```diff
@@ -60,21 +61,20 @@ def build_di_container(
 ):
     logger = getLogger()
     cwd = Path().resolve()
-    protostar_toml_path = search_upwards_protostar_toml_path(start_path=cwd)
-    project_root_path = (
-        protostar_toml_path.parent if protostar_toml_path is not None else cwd
-    )
-    protostar_toml_path = protostar_toml_path or project_root_path / "protostar.toml"
-
     configuration_file_factory = ConfigurationFileFactory(
         cwd, active_profile_name=active_configuration_profile_name
     )
     configuration_file = configuration_file_factory.create()
+    project_root_path = (
+        configuration_file.get_filepath().parent
+        if isinstance(configuration_file, FakeConfigurationFile)
```

Comment:
shouldn't this logic be encapsulated in `get_filepath()` ?

Reply:
[refactor project root path](https://github.com/software-mansion/protostar/pull/1066/commits/11dbc91bc0a6369b401ecb0eda6fdf31c513db86)